views/horizontal_nav.erb

```diff
@@ -0,0 +1,20 @@
+<%
+  path = request.path_info
+  nav = HorizontalNav.for(path)
```

Comment:
I love this change! This makes it easy for the horizontal nav to be reused because of your careful reorganizations of concerns and related logic.

Fantastic work, Erin!